src/main/java/seedu/address/logic/commands/ListIncompleteCommand.java

```diff
@@ -0,0 +1,18 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all incomplete tasks in the task book to the user.
+ */
+public class ListIncompleteCommand extends Command {
+
+    public static final String COMMAND_WORD = "listincomplete";
```

Comment:
Good Use of Clear Command Word. Using this is a great choice for clarity and readability.